src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -1100,6 +1167,12 @@ real(r8):: catanf                       ! hyperbolic temperature function from C
                   c = filter_soilc(fc)
                   decomp_k(c,j,i_cwd)   = k_frag  * t_scalar(c,j) * w_scalar(c,j) * depth_scalar(c,j) * &
                        o_scalar(c,j) * spinup_geogterm_cwd(c)
+! if use soil_matrix                  
```

Comment:
Align the comment with the block of code

Reply:
done